src/notebooks_service.py

```diff
@@ -223,8 +235,7 @@ def stop_notebook(user, namespace, project, commit_sha):
 
 
 @app.route(
-    urljoin(SERVICE_PREFIX, 'servers/<server_name>'),
-    methods=['DELETE']
+    urljoin(SERVICE_PREFIX, 'servers/<server_name>'), methods=['DELETE']
```

Comment:
keep old formating?

Reply:
no, this is automatically formatted using the style from `.style.yapf`